tests/receivers/smartagent/prometheus-exporter/testdata/httpd/metrics

```diff
@@ -0,0 +1,46 @@
+# HELP istio_agent_cert_expiry_seconds The time remaining, in seconds, before the certificate chain will expire. A negative    value indicates the cert is expired.
+# TYPE istio_agent_cert_expiry_seconds gauge
+istio_agent_cert_expiry_seconds{resource_name="default"} 7449418.02275500029
```

Comment:
I can look into fixing values equivalence checks to detect deviations (unless you'd like to in your translation progress?). Since these are static values it would hopefully be trivial to add them to the resource metrics.

Reply:
That’d be useful, sure! I don’t mean to impose on you though.